pkg/agent/metrics/prometheus.go

```diff
@@ -67,6 +67,33 @@ var (
 		Help:           "Flow count for each OVS flow table. The TableID is used as a label.",
 		StabilityLevel: metrics.STABLE,
 	}, []string{"table_id"})
+
+	OVSFlowOpsCount = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "antrea_agent_ovs_flow_ops_count",
+			Help:           "Number of OVS flow operations, partitioned by operations(add, modify and delete).",
```

Comment:
s/partitioned by operations(add, modify and delete)/partitioned by operation type (add, modify and delete)

same for the other places below

Reply:
Thank you for your comment. Fixed.